R/zzz.R

```diff
@@ -64,14 +64,13 @@ register_on_load <- function(pkg, ver = c(NA, NA), callback, message = NULL) {
 
 #' Check Suggested Dependency Against Version Requirements
 #'
-#' @eval typed_param(pkg, character(1L),
-#'   "A package name.")
-#' @eval typed_param(ver,
-#'   "any object of length 2L whose elements can be provided to
-#'   `numeric_version()`",
-#'   "A minimum and maximum (inclusive) version requirement for
-#'   interoperability. When `NA`, no version requirement is imposed. Defaults to
-#'   no version requirement.")
+#' @param pkg (`character[1]`)\cr
+#'   A package name.
+#' @param ver (any object of length 2 whose elements can be provided to
```

Comment:
ok if so complicated we don't put in parentheses. basically I would just put numeric or just vector here and then explain in the argument description.